src/classic/components/InfoCols/InfoCols.tsx

```diff
@@ -0,0 +1,101 @@
+import { ArrowRightIcon } from "@heroicons/react/24/outline"
+import { HomepageSectionWrapper } from "../HomepageSectionWrapper"
+import InfoColsProps from "~/common/InfoCols"
+
+const InfoColsHeader = ({
+  title,
+  subtitle,
+}: Pick<InfoColsProps, "title" | "subtitle">) => (
+  <div className="flex flex-col gap-4">
+    {subtitle && (
+      <p className="text-subtitle uppercase tracking-widest">{subtitle}</p>
+    )}
+    <h1 className="text-secondary text-5xl font-semibold leading-tight">
+      {title}
+    </h1>
+  </div>
+)
+
+const InfoBoxes = ({ infoBoxes }: Pick<InfoColsProps, "infoBoxes">) => {
+  const mdColsClass =
+    infoBoxes.length === 1 ? "md:grid-cols-1" : "md:grid-cols-2"
+
+  // Follows current behaviour: if there are 4 info boxes, xl screen should stay at 2 columns instead of going up to 3
+  const xlColsClass =
+    infoBoxes.length === 1
+      ? "xl:grid-cols-1"
+      : infoBoxes.length % 2 === 0
+        ? "xl:grid-cols-2"
+        : "xl:grid-cols-3"
+
+  return (
+    <div>
+      <div
+        className={`grid grid-cols-1 gap-4 justify-between ${mdColsClass} ${xlColsClass}`}
+      >
+        {infoBoxes.map((infoBox, idx) => (
+          <div
+            key={idx}
+            className="flex flex-col gap-4 p-6 items-center text-center md:items-start md:text-left"
+          >
+            <h3 className="text-3xl font-bold text-subtitle">
+              {infoBox.title}
+            </h3>
+            <p className="text-paragraph text-xl">{infoBox.description}</p>
+          </div>
+        ))}
+      </div>
+    </div>
+  )
+}
+
+const InfoColsFooter = ({
+  buttonLabel,
+  buttonUrl,
+}: Pick<InfoColsProps, "buttonLabel" | "buttonUrl">) => {
+  return (
+    buttonLabel &&
+    buttonUrl && (
+      <div className="text-lg font-semibold uppercase">
+        <a
```

Comment:
depending on the type of link (internal or extenal), this might need to accept NextLink as well
https://nextjs.org/docs/pages/api-reference/components/link